### PR TITLE
Add tutorial tweaks

### PR DESCRIPTION
**Pull request by JShep1**
Made some tweaks as I went through the tutorials.  Most of them were formatting and correcting a couple broken image links.

Signed-off-by: John Shepherd <john@openrobotics.org>

### CODE REVIEW

**Comment by JShep1** [comment]
I noticed that [here](https://github.com/ignitionrobotics/ign-gazebo/blame/master/tutorials/migrating_ardupilot_plugin.md#L485-L501)  The code makes mention of the old and new way of doing things, but there is no difference in the code, I tried to look for the replacement line but wasn't quite sure what this section of code was in the ignition equivalent.

File: tutorials/erb_template.md [hunk 1]
```diff
@@ -141,14 +141,13 @@ Instead of simple shapes, you can also use a nested loop to generate 100 actors
     </actor>
 
     <%
-        end
```
Comment (chapulina) [inline tutorials/erb_template.md:144]: Are you sure this needs to be removed? There are 2 for loops

**Review by chapulina — COMMENTED** [review]
Nice! I just have one little comment.

File: tutorials/erb_template.md (hunk 1 above)
Comment (JShep1) [reply tutorials/erb_template.md]: Oops!  Reverted here 1999945cdbfa42e06758f442da50625316f1e9e2

**Review by chapulina — APPROVED** [review]

**Pull request merged**